databricks/sdk/mixins/workspace.py

```diff
@@ -32,7 +32,6 @@ def list(self,
             for object_info in parent_list(path, notebooks_modified_after=notebooks_modified_after):
                 if recursive and object_info.object_type == ObjectType.DIRECTORY:
                     queue.append(object_info.path)
-                    continue
```

Comment:
this breaks established behavior, that thousands of projects depend on. Add an optional keyword argument disabled by default if you want this change